anti-analysis/anti-disasm/64-bit-execution-via-heavens-gate.yml

```diff
@@ -2,7 +2,7 @@ rule:
   meta:
     name: 64-bit execution via heavens gate
     namespace: anti-analysis/anti-disasm
-    author: '@recvfrom'
+    author: awillia2@cisco.com
```

Comment:
a brazen attempt at corporate marketing!

:-P

happy to accept these changes, too :-)